content/schedule.md

```diff
@@ -0,0 +1,11 @@
+{
+    "title": "Speakers and Schedule",
+    "description": "elm-conf will take place September 12, 2019 at the Stifel Theatre in St. Louis, MO as part of the Strange Loop preconference."
+}
+---
+
+# Speakers &amp; Schedule
+
+elm-conf will take place September 12, 2019 at the Stifel Theatre in beautiful downtown St. Louis, Missouri as part of the Strange Loop preconference.
+
+<a class="button" target="_blank" href="https://thestrangeloop.com/venue.html">Location and Directions</a>
```

Comment:
👀 Okay to link to Strange Loop's venue page?
💡 This button doesn't have the info icon yet.

Reply:
Let's leave it off. I don't think it needs an icon anymore.